src/component/EventDetails.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-lone-blocks */
+import { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { useNavigate } from 'react-router-dom';
+import { Button, Image, Table } from 'react-bootstrap';
+import { BiLeftArrow } from 'react-icons/bi';
+import { GiCancel } from 'react-icons/gi';
+import { RiReservedLine } from 'react-icons/ri';
+import { IoIosArrowDropright } from 'react-icons/io';
+import { deleteAttendance } from '../redux/event/event';
+import '../css/EventDetails.css';
+
+const EventDetails = () => {
+  const details = useSelector((state) => state.events.eventDetails);
+  const loading = useSelector((state) => state.events.loading);
+  const [date, setDate] = useState(null);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const deleteAttendanceAction = bindActionCreators(deleteAttendance, dispatch);
+
+  useEffect(() => {
+    if (details.date) {
+      const eventDate = new Date(details.date);
+      const months = {
+        '01': 'Jan',
+        '02': 'Feb',
+        '03': 'Mar',
+        '04': 'Apr',
+        '05': 'May',
+        '06': 'Jun',
+        '07': 'Jul',
+        '08': 'Aug',
+        '09': 'Sep',
+        10: 'Oct',
+        11: 'Nov',
+        12: 'Dec',
```

Comment:
- [ ] You should follow the same pattern as the above keys in this object, stringify all of them.

Reply:
Hi, @MuriloRoque! :wave:
Thanks for your detailed code review! :pray:
In this particular case, we won't be able to comply with your request because of linter requirements, though...
But we agree that there is a break of pattern that would look nicer if we could avoid it...